as/thrift.js

```diff
@@ -45,6 +45,7 @@ function TChannelAsThrift(opts) {
         'must pass source as an argument');
 
     self.thriftSource = opts.source;
+    self.thriftFileName = opts.thriftFileName || 'service.thrift';
```

Comment:
We can follow-up to support AsThrift({idls, entryPoint}) when ThriftRW supports that form of constructor. For now, this is a fine migration step.